src/sections/Projects/Sistent/components/select/section-nav.js

```diff
@@ -0,0 +1,38 @@
+import { navigate } from "gatsby";
+import TabButton from "../../../../../reusecore/Button";
+import React from "react";
+import { useLocation } from "@reach/router";
+export default function SectionNav() {
```

Comment:
what is use of this file? Routing is done in one of same file where you have defined component. Checkout existing components.

Reply:
The file is for the UI component of the section navigation. I extracted this so it can be reused for `code.js`, `guidance.js`, and `index.js` files.

![image](https://github.com/user-attachments/assets/e45a3ed1-d63d-4a00-b2f8-8a104f2a87b9)